driver/driver.mock.go

```diff
@@ -1,289 +1,656 @@
-// Code generated by MockGen. DO NOT EDIT.
-// Source: github.com/container-storage-interface/spec/lib/go/csi (interfaces: IdentityServer,ControllerServer,NodeServer)
```

Comment:
Please re-generate this once driver/driver.go has been replaced.

Reply:
Please let me know steps to re-generate it.